framework/src/parser/Parser.C

```diff
@@ -1236,6 +1236,14 @@ Parser::extractParams(const std::string & prefix, InputParameters & p)
           dynamic_cast<InputParameters::Parameter<std::vector<ptype>> *>(par),                     \
           in_global,                                                                               \
           global_params_block)
+#define setmap(key_type, mapped_type)                                                              \
```

Comment:
Just for good measure - maybe undef this with the other undefs below.

Reply:
:+1: